src/core/blockchain/data-loader.ts

```diff
@@ -166,22 +183,24 @@ export async function getAchievementsNFTConfig(
 }
 
 export async function getDocumentationTree(
-  communityId: String
+  communityId: String,
+  network: Network
 ): Promise<Documentation> {
-  const provider = await createRpcProvider();
-  const contract = new PeeranhaContentWrapper(provider);
-  return contract.getDocumentationTree(Number(communityId));
+  const provider = await createRpcProvider(network);
+  const contract = new PeeranhaContentWrapper(provider, network);
+  return contract.getDocumentationTree(communityId);
 }
 
 export async function getItemProperty(
   propertyId: number,
-  postId: number,
-  replyId?: number,
-  commentId?: number
+  postId: string,
+  network: Network,
```

Comment:
May be better to make it the last argument for consistency.

Reply:
So, i need to send the network argument explicitly ( getItemProperty(postId, network: network1) )?

Reply:
I see now that the remaining 2 are optional. All good then.